modular_skyrat/modules/morenarcotics/code/cocaine.dm

```diff
@@ -53,8 +53,17 @@
 	M.AdjustImmobilized(-15 * REM * seconds_per_tick)
 	M.AdjustParalyzed(-15 * REM * seconds_per_tick)
 	M.adjustStaminaLoss(-2 * REM * seconds_per_tick, 0)
+	M.adjustToxLoss(1 * REM * seconds_per_tick, 0)
```

Comment:
1 toxin per sec is alot, compared to nuka-cola which would be less effort, non-dangerous, easier to get, and faster to make en-masse

Reply:
i am less concerned with making cocaine a superior nuka cola than i am making what was ultimately supposed to be a roleplay drug in fact bad for you rather than having people say 'cocaine is harmless' IC

Reply:
then what do you suggest, seeing as otherwise it is a consequence free drug you would use for a speedboost. remove the speedboost?

Reply:
most of the drugs are consequence free, if not literally all of them shy of overdosing, including the ones from the new PR, singling one out to be giga-poison for the sake of stopping 'people saying it's harmless IC' just reverts back to 'something you would only ever use if you intend to powergame chem anyway'

Reply:
Could probably make it half of that amount of damage per second, and also replace that `0` with `FALSE` here, please.